Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -14,14 +14,18 @@ public class Game {
 	}
 
 	public func playGame() {
+		let maxNumberOfGuesses = guessManager.maxNumberOfGuesses
+		view.promptNumberOfGuesses()
+		let numberOfGuesses = view.receiveNumberOfGuesses(maxNumberOfGuesses)
```

Comment:
Sorry for my confusion, but the receiving `numberOfGuesses` to me is a function of the game because it requires the user to interact with the game. It is part of set up, which could be separate from player turn, but then I still need to call the setup method from something that is triggered by my `playGame()` method. I don't see a way that I can separate receiving `numberOfGuesses` from the running of the game because it involves user interaction.